docs/src/model_setup/coriolis.md

```diff
@@ -7,58 +8,73 @@ three options are available: no rotation, $f$-plane, and $\beta$-plane.
     Coriolis and centripetal forces, both of which arise in non-inertial reference frames. But here the model only
     considers the Coriolis force.
 
-To use no rotation, pass
-```
-coriolis = nothing
-```
+## No rotation
+
+By default there is no rotation. This can be made explicit by passing `coriolis = nothing` to a model constructor.
 
 ## $f$-plane
 
 To set up an $f$-plane with, for example, Coriolis parameter $f = 10^{-4} \text{s}^{-1}$
-```@example
-using Oceananigans # hide
-coriolis = FPlane(f=1e-4)
+
+```@meta
+DocTestSetup = quote
+    using Oceananigans
+end
+```
+
+```jldoctest
+julia> coriolis = FPlane(f=1e-4)
+FPlane{Float64}: f = 1.00e-04
 ```
 
 An $f$-plane can also be specified at some latitude on a spherical planet with a planetary rotation rate. For example,
 to specify an $f$-plane at a latitude of $\varphi = 45°\text{N}$ on Earth which has a rotation rate of
 $\Omega = 7.292115 \times 10^{-5} \text{s}^{-1}$
-```@example
-using Oceananigans # hide
-coriolis = FPlane(rotation_rate=7.292115e-5, latitude=45)
+
+```jldoctest
+julia> coriolis = FPlane(rotation_rate=7.292115e-5, latitude=45)
+FPlane{Float64}: f = 1.03e-04
 ```
+
 in which case the value of $f$ is given by $2\Omega\sin\varphi$.
 
 To set up an $f$-plane with non-traditional Coriolis terms, for example, with 
 $\bm{f} = (0, f_y, f_z) = (0, 2, 1) \times 10^{-4} \text{s}^{-1}$,
-```@example
-using Oceananigans # hide
-coriolis = NonTraditionalFPlane(fz=1e-4, fy=2e-4)
+
+```jldoctest
+julia> coriolis = NonTraditionalFPlane(fz=1e-4, fy=2e-4)
+Non-TraditionalFPlane{Float64}:fz = 1.00e-04, fy = 2.00e-04
```

Comment:
👏🏼

though I should say, missing a space after the colon

Reply:
Ah thanks for catching that!